antenna/ext/gcs/crashstorage.py

```diff
@@ -126,8 +126,9 @@ def verify_write_to_bucket(self):
     def check_health(self, state):
         """Check GCS connection health."""
         try:
-            # get the bucket to verify GCS is up and we can connect to it.
-            self.client.get_bucket(self.bucket)
+            # check if a blob exists to verify GCS is up and we can connect to it,
+            # without exceeding the permissions granted by roles/storage.objectUser
+            self.client.bucket(self.bucket).blob(generate_test_filepath()).exists()
```

Comment:
Will the test file always exist? I'm not sure I understand this test and what will happen in this try/except block that catches `Exception`.

Reply:
the test file won't exist, but that's fine because the exists function returns a boolean unless it lacks permissions